deploy/registration-service/registration-service.yaml

```diff
@@ -167,10 +167,6 @@ objects:
                     configMapKeyRef:
                       name: registration-service
                       key: verification.message_template
-              resources:
-                requests:
-                  cpu: "50m"
-                  memory: "100M"
```

Comment:
Why it's being deleted?

Reply:
I'm not sure, I don't see anything in my changes that would cause that.

Reply:
Then please bring it back :)

Reply:
The reason is that the resource requests are not defined in the registration-service repo.

Reply:
fixed in https://github.com/codeready-toolchain/registration-service/pull/163

Reply:
Good catch. BTW I think we don't really need that much resources (there are three replicas so the total would be 150m/300M) but let's leave it for another PR :)

Reply:
Thanks @MatousJobanek. @alexeykazakov it's back now after pulling in Matous' change and regenerating.